### PR TITLE
ANIMAL FARM 3: the one goof feature people actually wanted

### PR DESCRIPTION
**Pull request by Iamgoofball**
oh god kill me

### CODE REVIEW

**Comment by kevinz000** [comment]
freeze

**Comment by Iamgoofball** [comment]
i know we're in a freeze but getting this working is a month project so gotta see if people are sstill interested and also active maintainer feedback

File: code/__PATH_COMPATIBILITY/path_compatibility.dm [hunk 1]
```diff
@@ -18,4 +18,29 @@
...
+/mob/living/simple_animal/hostile/carp/holocarp
+	parent_type = /mob/living/simple_animal/farm/carp/holocarp
+/mob/living/simple_animal/hostile/carp/ranged
+	parent_type = /mob/living/simple_animal/hostile/carp_ranged
+/mob/living/simple_animal/hostile/carp/ranged/chaos
+	parent_type = /mob/living/simple_animal/hostile/carp_ranged/chaos
+/mob/living/simple_animal/cow
+	parent_type = /mob/living/simple_animal/farm/cow
+/mob/living/simple_animal/hostile/carp/cayenne
+	parent_type = /mob/living/simple_animal/farm/carp/cayenne
```
Comment (Cyberboss) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: Please, just repath!

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/__PATH_COMPATIBILITY/path_compatibility.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: no can do compadre until we don't have 50 maps

File: code/modules/mob/living/simple_animal/friendly/farm/animals/chicken.dm [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/mob/living/simple_animal/farm/chicken
+	name = "\improper chicken"
+	desc = "Hopefully the eggs are good this season."
+	icon_state = "chicken_white"
+	icon_living = "chicken_white"
+	icon_dead = "chicken_white_dead"
+	speak = list("Cluck!","BWAAAAARK BWAK BWAK BWAK!","Bwaak bwak.")
```
Comment (F-OS) [inline code/modules/mob/living/simple_animal/friendly/farm/animals/chicken.dm]: Add cluck cluck cluck, tuck tuck, ERR ERR ERR, and  Kuh-kuh-kuh-KACK! 

Source: Have chickens

**Comment by Armhulen** [comment]
**I SAID FREEZE**
I'm just getting a drink...

File: code/modules/mob/living/simple_animal/friendly/farm/animals/chicken.dm [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/mob/living/simple_animal/farm/chicken
+	name = "\improper chicken"
+	desc = "Hopefully the eggs are good this season."
+	icon_state = "chicken_white"
```
Comment (F-OS) [inline code/modules/mob/living/simple_animal/friendly/farm/animals/chicken.dm]: Suggestion, sprite brown, black, and yellow chickens, all chickens aren't white

**Comment by ExcessiveUseOfCobblestone** [comment]
Andrew Jackson, aka llabfoogmai, is a great coder and should be allowed back onto this git!

**Comment by ShadowDeath6** [comment]
If this pr does what I think it does why not give it to chef

**Comment by Iamgoofball** [comment]
botanists are farmers, chef is not

**Comment by ShadowDeath6** [comment]
Botany has enough stuff to fuck around with though since it's been powercreep central

**Comment by lzimann** [comment]
Freeze

File: code/__PATH_COMPATIBILITY/path_compatibility.dm (hunk 1 above)
Comment (Cyberboss) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: git grep -l ... | xargs sed -i -b 's/.../.../g'

**Comment by Iamgoofball** [comment]
@lzimann can you leave this open for the purposes of auto-updating the commits and review

File: code/__PATH_COMPATIBILITY/path_compatibility.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: ????

**Comment by kevinz000** [comment]
@Iamgoofball i think the point of a freeze is to not spend time reviewing features....

**Comment by Armhulen** [comment]
I thought it was to focus on cutting down on bugs,

**Comment by Cyberboss** [comment]
It defeats the purpose of people are still working on features

File: code/__PATH_COMPATIBILITY/path_compatibility.dm (hunk 1 above)
Comment (Haevacht) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: Either he smashed his face into the keyboard or he's speaking in regex and git at the same time.

**Pull request closed without merging**